### ISSUE TITLE
Code coverage is even worse than you describe

### ISSUE THREAD

**Issue by carols10cents** [laboon/ebook#68]
You say:

> Having a statement or method covered by tests does not mean that all defects in that unit of code have been found by the test!

But it's EVEN WORSE than that-- having a statement or method executed during tests does not even mean that you asserted on ANYTHING having to do with that code at all!!

Please instruct your students that they can easily win the adoration of managers demanding 100% method coverage AND passing tests simply by calling every method from tests but never asserting anything. I need more cohorts equipped with the knowledge to subvert bullshit developer metric targets.

**Comment by laboon** [comment]
YES.  I bring this up in class but overlooked in the book. Will emphasize (and probably use your "adoration of managers" example if you don't mind)

**Comment by carols10cents** [comment]
:heart: :heart: :heart:
